### PR TITLE
Videos: use WEB client instead of WEB CREATOR

### PR DESCRIPTION
**Pull request by unixfox**
This switch to using WEB client when a potoken is configured. Otherwise try with Android test suite if there is no potoken configured.

### CODE REVIEW

**Review by SamantazFox — APPROVED** [review]

**Review by syeopite — APPROVED** [review]

**Comment by unixfox** [comment]
Switching to a draft since after a while (5 minutes) all the requests get blocked. But I guess it's better than nothing since ANDROID_TEST_SUITE is straight blocked.

**Comment by unixfox** [comment]
On matrix, iOS is the only candidate for keeping a working condition without po_token.

**Comment by unixfox** [comment]
PR description updated

File: src/invidious/videos/parser.cr [hunk 1]
```diff
@@ -53,9 +53,9 @@ end
 def extract_video_info(video_id : String)
   # Init client config for the API
   client_config = YoutubeAPI::ClientConfig.new
-  # Use the WEB_CREATOR when po_token is configured because it fully only works on this client
+  # Use the WEB when po_token is configured
   if CONFIG.po_token
-    client_config.client_type = YoutubeAPI::ClientType::WebCreator
+    client_config.client_type = YoutubeAPI::ClientType::Web
   end
```
Comment (syeopite) [inline src/invidious/videos/parser.cr:59]: The default client type is already WEB so this section can be removed

Comment (unixfox) [reply src/invidious/videos/parser.cr:59]: doesn't this allow to explicitly say that we use WEB? I was thinking the same, though. I don't know.

Comment (syeopite) [reply src/invidious/videos/parser.cr:59]: I mean the logic will also default to the WEB client when the conditional fails. Its moot to include that.

Might be easier to just do 

```crystal
  if CONFIG.po_token
    client_config.client_type = YoutubeAPI::ClientType::Web
  else
    client_config.client_type = YoutubeAPI::ClientType::AndroidTestSuite
  end
```

As so we only have a single request. Though I guess the android client also doesn't return all of the data we need

**Review by syeopite — APPROVED** [review]

**Review by SamantazFox — APPROVED** [review]

**Comment by SamantazFox** [comment]
Thank you for reverting this, it was much needed!

**Pull request merged**